### ISSUE TITLE
Strange compatibility issue with Advanced Control Net nodes

### ISSUE THREAD

**Issue by ghostsquad** [jags111/efficiency-nodes-comfyui#133]
I started an issue over here: https://github.com/Kosinkadink/ComfyUI-Advanced-ControlNet/issues/82

The TLDR is that there's some strange compatibility issue with the Advanced ControlNet Nodes and the Efficiency Advanced KSampler. I have both sets of nodes up to date (as of writing this issue).

There's an optional `model` input/output on the Advanced ControlNet nodes (because I guess some controlnets use the model). I found that when that optional model input/output is _not_ connected, the ksampler appears to think nothing has changed, and doesn't regenerate the image. This only seems to happen with the Efficient KSampler. The image below (workflow embedded) shows that problem. All output images should be identical. But you can see that one of them (the one connected to Efficient KSampler and ControlNet without the model connected) remains as the previously generated image (when the cnet groups were muted).

![advanced-controlnet-issue-82-with-vanilla-ksamplers](https://github.com/jags111/efficiency-nodes-comfyui/assets/903488/ef236ecf-68bb-46fc-8f0c-3414c86c9bf2)
